packages/dioxus-tui/tests/events.rs

```diff
@@ -131,13 +134,9 @@ fn mouse_up() {
             modifiers: KeyModifiers::NONE,
         }));
         cx.render(rsx! {
-            div {
-                width: "100%",
-                height: "100%",
-                onmouseup: move |_| {
+            div { width: "100%", height: "100%", onmouseup: move |_| {
                     tui_ctx.quit();
-                },
-            }
+                } }
```

Comment:
It looks like you ran the `dx fmt` command and it is having a few issues?

Reply:
The workspace settings for vscode automatically run the built-in formatter, I have mine configured to [Prettier](https://prettier.io/). That might be the issue... Let me try to fix it

Reply:
I fixed this by manually overriding the workspace setting so it doesn't auto-format. I think it may be worth removing the workspace setting altogether though unless it specifies which formatter to use, otherwise it could create some thrashing depending on the contributor.

Reply:
Interesting! I'm didn't know prettier would try to format macros

Reply:
I've added a `dxfmt::skip` attribute to the parts that were causing issues. I've also updated the settings.json to specify a formatter and added some workspace recommendations. I can move these to a separate PR though if you think that bloats this one

Reply:
>I fixed this by manually overriding the workspace setting so it doesn't auto-format. I think it may be worth removing the workspace setting altogether though unless it specifies which formatter to use, otherwise it could create some thrashing depending on the contributor.

Rust analyzer or cargo fmt seems pretty standard for formatting. If a different formatter messes it up a bit, that is probably fine. The RSX formatting isn't consistent throughout the repo. The real solution here is to get `dx fmt` running in CI which should consistently. I'm not opposed to removing it, but I think it is fine for most people. For anything outside of macros, the CI will catch it

Reply:
Yeah i am going to revert the settings changes for this PR. There's some weird inconsistencies with how the formatting is being applied, I'll put that as a follow-up issue